config/samples/sfptpd/README.md

```diff
@@ -0,0 +1,66 @@
+
+# Solarflare Enhanced PTP Daemon (sfptpd)
+
+To run `sfptpd` in a Kubernetes cluster, deploy the `sfptpd` privileged container to the node receiving PTP signal.
+
+This directory provides Kustomize formatted:
+
+* base ([DaemonSet](base/daemonset.yaml)),
+* components (eg. [MachineConfig for chronyd](chronyd/machineconfig-4.8.0/99-worker-chronyd.yaml), [ConfigMap *sfptpd.conf*](conf/ptp1/configmap.yaml)), and
+* example overlay.
+
+(The Onload Operator does not presently manage `sfptpd`; please inform your AMD account manager know if this would be a valued feature addition.)
+
+## Quickstart
+
+The example Kustomize overlay [default-ocp/kustomization.yaml](default-ocp/kustomization.yaml) is tailored to OpenShift's MachineConfig and will template the following:
+
+1. A DaemonSet deploying the `sfptpd` container, specifying the pod's PTP interface name as `sf0` and using a *sfptpd.conf* file specifying `ptp1` kernel device.
```

Comment:
Where's this `ptp1`? The `ptp1` specified in the config file is scoped only to the config file, it is the name of a 'sync instance'. The `/dev/ptp*` kernel devices I believe are already mapped into the pod?